### PR TITLE
Improve look of the user email settings page

### PR DESCRIPTION
**Pull request by gittex**
This PR tries to improve the look and feel of the email settings page.

It aligns the design to the other settings pages, adds highlightning of the current item
and introduces sorting of the email adresses listed. The primary address will always be listed first with the
others following in lexical order.

### CODE REVIEW

**Comment by phsmit** [comment]
Looks much better indeed :+1:

**Comment by phsmit** [comment]
About the sort, wouldn't it be better to have the initial sorting done by the database? (adding .Asc() to the query). After that we can indeed move the primary email to the top

**Comment by gittex** [comment]
Doing the sort would at least make the code look a little cleaner.

But since I have no experience with the ORM, yet I did stick to what I know.
The way it is implemented now would allow for easy addition of more sphoisticated sorting orders, but if you insist I will change the patch to do the sort in the DB.

**Comment by phsmit** [comment]
I'm not a Gogs maintainer, so I will not insist anything :)

**Comment by gittex** [comment]
I've refactored my patch a little to touch less existing code.

File: conf/locale/locale_en-US.ini [hunk 1]
```diff
@@ -219,12 +219,13 @@ change_password_success = Password is changed successfully. You can now sign in
```
Comment (Unknwon) [inline conf/locale/locale_en-US.ini]: Please rule out this file since merged in another PR.

File: models/user.go [hunk 2]
```diff
@@ -629,7 +649,7 @@ func GetUserIdsByNames(names []string) []int64 {
 // Get all email addresses
 func GetEmailAddresses(uid int64) ([]*EmailAddress, error) {
 	emails := make([]*EmailAddress, 0, 5)
-	err := x.Where("uid=?", uid).Find(&emails)
+	err := x.Where("uid=?", uid).Asc("Email").Find(&emails)
```
Comment (Unknwon) [inline models/user.go]: In ORM you need `email` not `Email`.

File: public/ng/css/gogs.css [hunk 3]
```diff
@@ -1813,6 +1813,7 @@ The register and sign-in page style
 #repo-hooks-history-panel,
```
Comment (Unknwon) [inline public/ng/css/gogs.css]: Hmm... you should change `.less` file not the css ones.

**Comment by Unknwon** [comment]
Thanks your PR!

I've made some comments, ping me if you run into any problem!

File: models/user.go (hunk 2 above)
Comment (lunny) [inline models/user.go]: The default Mapper rule of xorm is `EmailName <-> email_name`.

**Comment by Unknwon** [comment]
Close due to https://github.com/gogits/gogs/pull/895 , thanks your PR, though!

**Pull request closed without merging**